src/nlp/builder.py

```diff
@@ -344,7 +348,9 @@ def download_and_prepare(
             download_config.cache_dir = os.path.join(self._cache_dir_root, "downloads")
             download_config.force_download = download_mode == FORCE_REDOWNLOAD
 
-            dl_manager = DownloadManager(dataset_name=self.name, download_config=download_config,)
+            dl_manager = DownloadManager(
+                dataset_name=self.name, download_config=download_config, data_dir=self.config.data_dir
```

Comment:
@thomwolf - the DownoadManager has a nice property called `manual_dir` which refers to the `data_dir` the `data_dir` of the DownloadManager was previously never set to any value so I change that here and set it equal to `self.config.data_dir`.